File: 02_EstructurasControl/Repetitivas/For/C#/Ejemplo-01/Ejemplo01.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _1_For
{
    internal class Program
    {
        static void Main(string[] args)
        {
            //Pide al usuario un número y genera su tabla de multiplicar hasta 10.
            Console.WriteLine("Ingrese un número para generar su tabla de multiplicar:");
            int numero = int.Parse(Console.ReadLine());
            for (int i = 1; i <= 10; i++)
            {
                int resultado = numero * i;
                Console.WriteLine($"{numero} x {i} = {resultado}");
            }
            Console.ReadKey();
        }
    }
}

File: 02_EstructurasControl/Repetitivas/For/Python/Ejemplo01.py
# Pedir al usuario un número para generar su tabla de multiplicar
numero = int(input("Ingrese un número para generar su tabla de multiplicar: "))

# Generar la tabla de multiplicar hasta el 10
for i in range(1, 11):  # range(1, 11) genera números del 1 al 10
    resultado = numero * i
    print(f"{numero} x {i} = {resultado}")
